examples/aws.md

```diff
@@ -37,3 +45,7 @@ upstreams:
   * `autoscaling_group` – The name of the corresponding Auto Scaling group. Use of wildcards is supported. For example, `backend-*`.
   * `port` – The port on which our backend applications are exposed.
   * `kind` – The protocol of the traffic NGINX Plus load balances to the backend application, here `http`. If the application uses TCP/UDP, specify `stream` instead.
+  * `max_conns` – The maximum number of simultaneous active connections to an upstream server.
```

Comment:
Maybe we can add the defaults to `max_conns`, `fail_timeout` and `max_fails` in the doc? Something like:

* `max_conns` – The maximum number of simultaneous active connections to an upstream server. Default value is `0`, meaning there is no limit.
 * `max_fails` – The number of unsuccessful attempts to communicate with an upstream server that should happen in the duration set by the `fail-timeout` to consider the server unavailable. Default value is `1`. The zero value `0` disables the accounting of attempts.
  * `fail_timeout` – The time during which the specified number of unsuccessful attempts to communicate with an upstream server should happen to consider the server unavailable. Default value is `10s`